prisma/schema.prisma

```diff
@@ -127,6 +127,34 @@ model Claim {
   lastUpdatedAt DateTime @default(now())
 }
 
+// this is for validator service
+model CandidUserInfo {
+  id         Int    @id @default(autoincrement())
+  claimId    Int?   @unique
+  firstName  String
+  lastName   String
+  email      String
+  profileURL String
+}
+
+model Validation {
+  id        String   @id @default(uuid())
+  createdAt DateTime @default(now())
+
+  contex          String
+  validatorEmail   String
+  claimId          Int
+  validationStatus ValidationStatus @default(PENDING)
+  response         Boolean?         @default(false)
+  validationDate   DateTime?
+  statement        String?
+}
+
+enum ValidationStatus {
+  PENDING
+  COMPLETED
+}
```

Comment:
should we have 'REJECTED' or 'ABANDONED' too?

Will the email have an option to like say ' don't bother me i don't want to do this' without making an actual claim?

Reply:
to me "PENDING" means we could bother the guy again and send him reminder maybe

but ABANDONED means no, too much time passed

and REJECTED means don't bother this guy!

Reply:
Good point i will add that and yeah sure we should consider rejecting